app/models/owner.js

```diff
@@ -18,8 +18,8 @@ export default VcsEntity.extend({
   name: attr('string'),
   login: attr('string'),
   channels: attr(),
-  isSyncing: attr('boolean'),
-  syncedAt: attr(),
+  isSyncing: attr(''),
+  syncedAt: attr(''),
```

Comment:
Does `attr('')` have a different effect than `attr()`? If not, then let's change these to `attr()` for consistency, it seems to be what we use more often

Reply:
@sorta Great finding. Thank you. I already corrected it.